run_iwyu_tests.py

```diff
@@ -83,26 +83,29 @@ def setUp(self):
       'prefix_header_includes_remove.cc': ['--prefix_header_includes=remove'],
       'prefix_header_operator_new.cc': ['--prefix_header_includes=remove'],
     }
-    prefix_headers = [self.Include('prefix_header_includes-d1.h'),
+    prefix_headers = ['-I .', 
+                      self.Include('prefix_header_includes-d1.h'),
                       self.Include('prefix_header_includes-d2.h'),
                       self.Include('prefix_header_includes-d3.h'),
                       self.Include('prefix_header_includes-d4.h')]
     clang_flags_map = {
-      'alias_template.cc': ['-std=c++11'],
+      'alias_template.cc': ['-I .', '-std=c++11'],
       'auto_type_within_template.cc': ['-std=c++11'],
       # MSVC targets need to explicitly enable exceptions, so we do it for all.
-      'catch.cc': ['-fcxx-exceptions', '-fexceptions'],
-      'clmode.cc': ['--driver-mode=cl', '/GF', '/Os', '/W2'],
-      'conversion_ctor.cc': ['-std=c++11'],
+      'catch.cc': ['-I .', '-fcxx-exceptions', '-fexceptions'],
+      'clmode.cc': ['-I .', '--driver-mode=cl', '/GF', '/Os', '/W2'],
+      'conversion_ctor.cc': ['-I .', '-std=c++11'],
       'deleted_implicit.cc' : ['-std=c++11'],
       'lambda_fwd_decl.cc': ['-std=c++11'],
-      'lateparsed_template.cc': ['-fdelayed-template-parsing'],
+      'lateparsed_template.cc': ['-I .', '-fdelayed-template-parsing'],
       'ms_inline_asm.cc': ['-fms-extensions'],
       'prefix_header_attribution.cc': [
-          self.Include('prefix_header_attribution-d1.h')],
+          '-I .', self.Include('prefix_header_attribution-d1.h')],
       'prefix_header_includes_add.cc': prefix_headers,
       'prefix_header_includes_keep.cc': prefix_headers,
       'prefix_header_includes_remove.cc': prefix_headers,
+      'new_header_path_local.cc': [''], # no include paths
```

Comment:
The problem here is adding `-I .` manually (it is not really a default value if you have to spell it out explicitly). Another problem is perception. For me at cursory glance it looks broken and diverts attention. It's not an issue of correctness but of readability and understandability. Dealing with subjective issues can be frustrating but we cannot brush them away.